### DISCUSSION TITLE
Login Issues

### DISCUSSION THREAD

**Discussion by kareemassad** [SteveG/underthebar#3] [Q&A] [answered]
Hello! Thank you for making this app, it's exactly what I've been looking for. I was wondering how I could best login when I typically login using "Login with Google". I have tried using the username under `/user/username` in the profile url such as this [https://www.hevy.com/user/username](https://www.hevy.com/user/username) but couldn't get it to work. Any ideas?

Thanks!
Kareem

**Comment by kareemassad** [comment, marked as answer]
Update: Issue fixed! Reset password and used email instead of username.

**Reply by SteveG** [reply]
second time this has been asked actually, great that you resolved. So the trick is to get a username and password by using the hevy password reset function?

**Reply by kareemassad** [reply]
Sorry for the late reply, yep that is exactly it. Currently, if there are too many failed attempts logging into their system, you will be locked out for some amount of time.
